src/lib/components/BlogPost.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
```

Comment:
This component will render a blog from the structured files in the `data` directory